### PR TITLE
refactor(infra): CDK stack code cleanup

### PR DESCRIPTION
**Pull request by imincik**
Make CDK code more readable and little better organized before we add lot of new resources.

Based on PR #47

### CODE REVIEW

**Comment by imincik** [comment]
@l0b0 , I need to review and merge this before #56 (because #56 depends on this PR). Are you able to have a look please ? Thanks

File: infra/datalake/datalake_stack.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                f"{endpoint}-endpoint-function",
+                function_name=f"{endpoint}-endpoint",
+                handler=f"endpoints.{endpoint}.entrypoint.lambda_handler",
+                runtime=aws_lambda.Runtime.PYTHON_3_6,
+                code=aws_lambda.Code.from_asset(
+                    path="..",
+                    bundling=core.BundlingOptions(
+                        # pylint:disable=no-member
+                        image=aws_lambda.Runtime.PYTHON_3_6.bundling_docker_image,
+                        command=["backend/endpoints/bundle.bash", f"{endpoint}"],
```
Comment (l0b0) [inline infra/datalake/datalake_stack.py:85]: `f"{endpoint}"` can be just `endpoint`

Comment (imincik) [reply infra/datalake/datalake_stack.py:85]: ah yes, thanks.

Comment (imincik) [reply infra/datalake/datalake_stack.py]: Fixed in c30b2b91eb9adc7bf5a4ef2a8cfc99d59bd5cc94

**Review by l0b0 — APPROVED** [review]

**Pull request merged**